theme/int/views/authentication/home.hbs

```diff
@@ -29,15 +29,143 @@
         <!-- GHOST (will be replaced by js) -->
         <div class="hidden" id="SC_THEME">int</div>
         <div class="int-blog">
+            <div class="content" style="opacity: 1; transition: opacity 0.7s ease-in;">
+                <div class="container">
+                    <div class="row section-landing flex-container">
+                        <div class="col-sm-6">
+                            <div class="pull-left">
+                                <h6 class="font-weight-bold">HPI Schul-Cloud International</h6>
+                                <h1 class="mb-3">Lernen global vernetzt<br>- digital unterstützt</h1>
+                                <a class="btn btn-primary mb-3 mb-md-0 hidden-sm-down" href="https://blog.hpi-schul-cloud.de/community">Jetzt registrieren</a>
+                            </div>
+                        </div>
+                        <div class="col-sm-6">
+                            <div class="text-xs-center text-md-right">
+                                <img class="mb-3 mb-md-0" src="/images/home/hero.svg" alt="Hero">
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="container">
```

Comment:
Please consider using `<section>` instead of `div`, if the content is semantically an independent group. 
Indicators: 
- The div contains its own headline.
- Multiple new lines in the code separate two `div`s from each other.

Reply:
👍 But again I am not sure whether I should change this because it's the current production implementation and the ticket just aims to change some small things.